clusters/app.ci/supplemental-ci-images/validation-images/shellcheck/shellcheck.yaml

```diff
@@ -1,38 +1,43 @@
-apiVersion: v1
-kind: List
```

Comment:
also, deleting the list and using the `---` yaml approach its not an ideal change.